src/modules/auto-copy-on-click.js

```diff
@@ -0,0 +1,53 @@
+/**
+ *  @author Deux Huit Huit
+ *
+ */
+(function ($, undefined) {
+	
+	'use strict';
+
+	var site = $('#site');
+	var sels = {
+
+		toCopy: '.js-text-to-copy',
+		copyBtn: '.js-copy-on-click-btn',
+		message: '.js-confirm-message'
+	};
+	var timer = null;
+
+	var onClick = function () {
+		var t = $(this);
+		var parent = t.closest('.js-copy-ctn');
+		var message = parent.find(sels.message);
+		var delay = t.attr('data-message-delay');
+		
+		parent.find(sels.toCopy).each(function () {
+			$(this).select();
+			document.execCommand('copy');
+		});
```

Comment:
This will not work as indented. Only the last element will be in the clipboard. Are we sure this is the behaviour we want ? I think that if parent.find(sels.toCopy) returns:

1. 0 element, it should log it and bail out.
1. 1 element, it should do it
1. More than one element, it should log it and either a) do nothing or b) copy the FIRST element (not the last like it is now)